doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -1021,3 +1022,17 @@
     // after
     #include "iox/duration.hpp"
     ```
+46. Renaming `byte_t` to `byte`
+    `byte` is not a simple type alias, now it is a distinct type like c++17 `std::byte`.
+    The type `byte` does not support any arithmetic operations nor has member functions.
+
+    ```cpp
+    //before
+    using byte_t = uint8_t;
```

Comment:
This line can be removed since it is part of the implementation not how it is used